modules/internal/DefaultRectangular.chpl

```diff
@@ -151,6 +157,97 @@ module DefaultRectangular {
                ignoreRunning = dataParIgnoreRunningTasks,
                minIndicesPerTask = dataParMinGranularity,
                offset=createTuple(rank, idxType, 0:idxType))
+      where tag == iterKind.standalone && !localeModelHasSublocales {
+      if debugDefaultDist {
+        writeln("*** In domain standalone code:");
+      }
+      const numTasks = if tasksPerLocale == 0 then here.maxTaskPar
+                       else tasksPerLocale;
+      if debugDefaultDist {
+        writeln("    numTasks=", numTasks, " (", ignoreRunning,
+                "), minIndicesPerTask=", minIndicesPerTask);
+      }
+      const (numChunks, parDim) = if __primitive("task_get_serial") then
+                                  (1, -1) else
+                                  _computeChunkStuff(numTasks,
+                                                     ignoreRunning,
+                                                     minIndicesPerTask,
+                                                     ranges);
+      if debugDefaultDist {
+        writeln("    numChunks=", numChunks, " parDim=", parDim,
+                " ranges(", parDim, ").length=", ranges(parDim).length);
+      }
+
+      if debugDataPar {
+        writeln("### numTasksPerLoc = ", numTasks, "\n" +
+                "### ignoreRunning = ", ignoreRunning, "\n" +
+                "### minIndicesPerTask = ", minIndicesPerTask, "\n" +
+                "### numChunks = ", numChunks, " (parDim = ", parDim, ")\n" +
+                "### nranges = ", ranges);
+      }
+
+      if numChunks <= 1 {
+        if rank == 1 {
+          for i in ranges(1) {
```

Comment:
Oh, shoot.  I thought I did do that, but I guess it got backed out before I tested and committed.  Since I have your blessing as is, I'd like to merge this, then clean that up and retest.